### PR TITLE
translations for the point data related texts

### PR DESCRIPTION
**Pull request by emre2038**
## Description
adding `i18n` class to necessary divs to be able to update point data related texts using the profile configuration

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-74

## How is it tested automatically?
created translation.feature

## How should a reviewer test it locally
* pick a translatable key from https://app.gitbook.com/o/-Ly4_0FbvQFY-m3TOzuc/s/-M3zapxOsLjq_qKRcmH8/profile-configuation#translatable-keys
* set it in the profile config `translations` property
* on the website confirm that the text is changed

## Screenshots
before : 
image

after :
image

### Added
* `__tests__/gui/translation.feature` to test the feature

### Updated
* `src/js/webflow/import.js` to add `i18n` class to the necessary elements

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

**Review by michaelglenister — COMMENTED** [review]
I added some comments

Seeing all this test data has made me think about a profile data factory

File: src/index.html [hunk 1]
```diff
@@ -638,7 +638,7 @@ <h1 class="clear">Map boundary change</h1>
                   </div>
                 </div>
               </div>
-              <p class="point-filters__no-data hidden">No filters available for the selected data.</p>
+              <p class="point-filters__no-data hidden i18n" data-i18n="No filters available for the selected data.">No filters available for the selected data.</p>
```
[the remark this answers is not in the document]
Comment (emre2038) [reply src/index.html:641]: yes I was considering if we should do that also. so if you also think that's what we should do, we can do it. but if we do it now, because of [WNCM-523](https://wazimap.atlassian.net/browse/WNCM-523) this would look _more_ broken since the texts will be empty when navigating to another geography.

File: src/index.html [hunk 2]
```diff
@@ -3741,7 +3741,7 @@ <h4>Indicator chart title</h4>
 /*------------------------------------- Hide webflow items on deploy */
 .disabled {
   pointer-events: none;
-}
+}
```
[the remark this answers is not in the document]
Comment (emre2038) [reply src/index.html:3744]: I just import the webflow files using the `import-webflow` script. not sure why it is adding random empty spaces

File: src/js/webflow/import.js [hunk 3]
```diff
@@ -1,19 +1,64 @@
...
-  headContent.appendChild(tag);
+function addStylesheet(window, path) {
+    const tag = window.document.createElement("link");
+    tag.setAttribute("rel", "stylesheet");
+    tag.setAttribute("href", path);
+    let headContent = window.document.getElementsByTagName('head')[0];
+    headContent.appendChild(tag);
 }
 
 function add_js(window, path) {
```
[the remark this answers is not in the document]
Comment (emre2038) [reply src/js/webflow/import.js:9]: done 👍

**Review by michaelglenister — APPROVED** [review]
Thanks, looks good
